jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -125,16 +125,20 @@ public boolean removeJob(JobHandle jobHandle) {
 	
 	public TimerJobInstance getTimerByName(String jobName) {
 		for (Timer timer : timerService.getTimers()) {
-			Serializable info = timer.getInfo();
-			if (info instanceof EjbTimerJob) {
-				EjbTimerJob job = (EjbTimerJob) info;
-				
-				EjbGlobalJobHandle handle = (EjbGlobalJobHandle) job.getTimerJobInstance().getJobHandle();
-				if (handle.getUuid().equals(jobName)) {
-					logger.debug("Job  {} does match timer and is going to be returned", jobName);
-					return handle.getTimerJobInstance();
-				}
-			}
+		    try {
+    			Serializable info = timer.getInfo();
+    			if (info instanceof EjbTimerJob) {
+    				EjbTimerJob job = (EjbTimerJob) info;
+    				
+    				EjbGlobalJobHandle handle = (EjbGlobalJobHandle) job.getTimerJobInstance().getJobHandle();
+    				if (handle.getUuid().equals(jobName)) {
+    					logger.debug("Job  {} does match timer and is going to be returned", jobName);
+    					return handle.getTimerJobInstance();
+    				}
+    			}
+		    } catch (NoSuchObjectLocalException e) {
+                logger.debug("Timer info for {} was not found ", timer);
```

Comment:
I think we could look over all log statements in this class  as part of another jira and fix in master branch and then propagate to other branches if changes are needed. Feel free to raise a Jira for this please but i think here its fine as it mimics master atm.